UnitTests/BraintreeSEPADirectDebitTests/BTSEPADirectDebitClient_Tests.swift

```diff
@@ -457,50 +457,6 @@ class BTSEPADirectDebitClient_Tests: XCTestCase {
         }
     }
     
-    func testTokenize_handleCreateMandateReturnsInvalidURL_returnsError_andSendsAnalytics() {
```

Comment:
This test is failing because it needs to run on iOS 12 which is not supported by out CI. Removing this test.